addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3416,3 +3416,39 @@ def test_out_invoice_reverse_caba(self):
                 'credit': value['debit'],
             })
         self.assertRecordValues(reversed_caba_move.line_ids, expected_values)
+
+    def test_tax_grid_remove_tax(self):
+        # Add a tag to tax_sale_a
+        tax_line_tag = self.env['account.account.tag'].create({
+            'name': "Tax tag",
+            'applicability': 'taxes',
+            'country_id': self.company_data['company'].country_id.id,
+        })
+
+        repartition_line = self.tax_sale_a.invoice_repartition_line_ids.filtered(
+            lambda x: x.repartition_type == 'tax')
```

Comment:
Imo, this is more elegant:
```suggestion
        repartition_line = self.tax_sale_a.invoice_repartition_line_ids\
            .filtered(lambda x: x.repartition_type == 'tax')
```
What do you think? (ignore this comment if you don't agree)